10-JavaScript/exercises/extend-clock.md

```diff
@@ -6,10 +6,10 @@ Earlier you used events & jQuery selectors to build a JavaScript powered digital
 
 Add 3 buttons with text showing the names of colors.  Add an `onClick` event to the buttons which changes the text color of the clock to match the button.  An effective strategy would be to create css classes and use jQuery to switch the class the clock is styled with.
 
-## 24-hour vs 12-hour clocks
+## Changing Font & Alignment
 
-Next add a 12-hour button which changes the clock to display in 12-hour time like 5:05:45pm, and a 24-hour button which changes the clock to display in 24-hour mode like 17:05:45.
+Next with a pair of buttons change the clock's font and alignment to the left or right, in the browser.  
```

Comment:
The sentence on line 11 is not very clear to me. Maybe something like:

Next, add a pair of buttons, one with the text "left", and one saying "right". When the "right" button is clicked, the text of the clock should align left, and when the right button is clicked it should align right.

Did you intend to include something about font here too?